docs/src/test-runners-csharp.md

```diff
@@ -98,6 +98,57 @@ By default NUnit will run all test files in parallel, while running tests inside
 
 For CPU-bound tests, we recommend using as many workers as there are cores on your system, divided by 2. For IO-bound tests you can use as many workers as you have cores.
 
+## Using Verbose API Logs with NUnit
+
+If you want to use the [verbose API log](./debug.md#verbose-api-logs), via the `DEBUG` environment variable, you can route the logging data to the NUnit's _progress_ stream by using the provided `NUnitLogger`. To do this, you need to configure Playwright for .NET to use the logger. An efficient way of doing this is to create a _global_ setup fixture (essentially a class with a `[SetUpFixture]` attribute, in the root namespace of your test project) and create a method that is decorated with [`OneTimeSetUp`](https://docs.nunit.org/articles/nunit/writing-tests/attributes/onetimesetup.html). There, you can call `PlaywrightTest.ConfigureLogging()`. When nothing is passed through, a default logger will be created that outputs to the NUnit's Progress Output (visible in the `Test Output` pane in Visual Studio). Alternatively, you can provide a custom `ILoggerFactory` and provide a custom output for the diagnostics data (i.e. using a `ConsoleLogger`).
```

Comment:
Consider linkifying ILoggerFactory for those who don't know what the factory is.